test/e2e/generator/generate.go

```diff
@@ -189,7 +189,7 @@ func generateTestnet(r *rand.Rand, opt map[string]interface{}) (e2e.Manifest, er
 			nextStartAt += 5
 		}
 		node := generateNode(r, e2e.ModeFull, startAt, manifest.InitialHeight, false)
-		node.QueueType = manifest.QueueType
```

Comment:
How is this line related to adding different Tx sizes?

Reply:
It's a minor change that I thought doesn't warrant it's own PR. I just commented in the description:
> This also simplifies queue type to a testnet wide config (instead of a per node config)